content/workload-reduction-toolkit/identify-workload-issues/prioritise-change-using-impact-graphs.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Prioritise change using impact graphs
+colour: pink
+---
+
+# Prioritise change using impact graphs
+
+<strong class="govuk-tag">Template</strong>
+
+Impact graphs are used to measure the impact of different tasks and prioritise
+change. This template is designed for use by senior leadership teams in staff
+one-to-ones, or in team or whole staff meetings.
+
+<div class="govuk-grid-row dfe-width-container govuk-!-padding-bottom-6">
+  <div class="govuk-grid-column-full">
+    <div class="info-box">
+      <div class="info-box__corner">
+        <img src="/assets/images/clipboard-icon.svg" alt="Clipboard icon">
```

Comment:
this should be the download icon, like in:
https://improve-workload-and-wellbeing-for-school-staff.education.gov.uk/workload-reduction-toolkit/address-workload-issues/feedback-and-marking/review-marking-practice-with-an-impact-matrix/